README.md

```diff
@@ -53,7 +61,10 @@ In one sentence, explain why.
 
 ## A1.2
 
-«replace this with your answer»
+O(n)
+If we parse this, say using a stack, we will push until we get to the middle value,
+from which we will pop all the repeated values when we come across them meaning we only
+have to store n/2 of the elements, which is of O(n) complexity.
```

Comment:
# 3

Yes, although there's some discussion about how much we have to push on the stack, because we don't know when we have reached the middle.